tgui/packages/tgui/interfaces/Vending.tsx

```diff
@@ -1,11 +1,12 @@
 import { classes } from 'common/react';
 import { useBackend, useLocalState } from 'tgui/backend';
-import { Box, Button, Icon, LabeledList, NoticeBox, Section, Stack, Table } from '../components';
-import { Window } from '../layouts';
+import { Box, Button, Icon, LabeledList, NoticeBox, Section, Stack, Table } from 'tgui/components';
+import { Window } from 'tgui/layouts';
 
 type VendingData = {
   onstation: boolean;
-  department: string;
+  department_bitflag: string;
+  jobDiscount: number;
```

Comment:
I've ported the jobDiscount portion as jlsnow cleaned up the duplicated code. 

Its meant to take an inputted decimal (on tg defined as 0.2) and convert it into a percentage as a discount based on department affiliation.

Given that there is no discount being inputted. It doesn't make a difference.